backends/python/server/text_embeddings_server/models/flash_bert.py

```diff
@@ -1,50 +1,95 @@
 import torch
-
 from pathlib import Path
 from torch import nn
+import torch.nn.functional as F
 from typing import Type, List
 from safetensors import safe_open
 from transformers.activations import ACT2FN
 from transformers.models.bert import BertConfig
 from opentelemetry import trace
-
-# Flash attention imports
-import dropout_layer_norm
-
 from text_embeddings_server.models import Model
 from text_embeddings_server.models.types import FlashBatch, Embedding
 from text_embeddings_server.utils.flash_attn import attention
+from text_embeddings_server.utils.device import use_ipex
 
 tracer = trace.get_tracer(__name__)
 
 
+def hpu_add_layer_norm(
+    add: torch.Tensor,
+    x: torch.Tensor,
+    weight: torch.Tensor,
+    bias: torch.Tensor,
+    epsilon: float,
+    add_back: bool,
+):
+    if add is not None:
+        added_tensor = torch.add(add, x, alpha=1.0)
+        output = F.layer_norm(added_tensor, [x.size(-1)], weight, bias, epsilon)
+        if add_back:
+            add.add_(x)
+        return output
+    else:
+        return F.layer_norm(x, [x.size(-1)], weight=weight, bias=bias, eps=epsilon)
```

Comment:
I think would make sense to move this to a file like and import it in the case of hpu device only, same as `import dropout_layer_norm`

Reply:
Can I create a python file `hpu_op.py` under the path `backends/python/server/text_embeddings_server/utils/` and put the function of `hpu_add_layer_norm` in this file?

Reply:
sorry for the late response, I was ooo. looking back at it, this method only depends on torch and is generic enough so no need for an hpu ops file.